### ISSUE TITLE
False Positive: Php reserved $http_response_header variable is not set

### ISSUE THREAD

**Issue by simonberger** [kalessil/phpinspectionsea#1628]
PHP Language level: 5.3 (shouldn't  matter)

$http_response_header is reported with `seems to be not defined in the scope`
But it is a kind of PHP global variable.
https://www.php.net/manual/en/reserved.variables.httpresponseheader.php

**Comment by rentalhost** [comment]
Confirmed.

```php
function test()
{
    isset($http_response_header);
}
```

**Comment by kalessil** [comment]
Fixed in #1629

**Comment by dryabov** [comment]
@kalessil Why other superglobals are resolved properly, and these ones aren't?
And by the way, rarely used `$argc` and `$argv` are not resolved too.

PS. I was sure PhpStorm reads superglobals from the `_superglobals.php` stub file, but now I'm doubtful about it (because it highlights `$argc`, `$argv`, and `$HTTP_RAW_POST_DATA` as undefined).
